qiskit_optimization/algorithms/recursive_minimum_eigen_optimizer.py

```diff
@@ -135,7 +135,7 @@ def __init__(self, min_eigen_optimizer: MinimumEigenOptimizer, min_num_vars: int
         variables is reached.
 
         Args:
-            min_eigen_optimizer: The eigen optimizer to use in every iteration.
+            optimizer: The optimizer to use in every iteration.
             min_num_vars: The minimum number of variables to apply the recursive scheme. If this
                 threshold is reached, the min_num_vars_optimizer is used.
             min_num_vars_optimizer: This optimizer is used after the recursive scheme for the
```

Comment:
I know this was not changed, but perhaps we should state in the docstring that internally this defaults to the MinEigOpt.. when None.

Reply:
Good point. Added. Also I reviewed the docstrings, there a few updates.